docs: add troubleshooting entry for ComfyUI --normalvram error after upgrade (#3328)
beclab/Olares: docs/use-cases/comfyui-common-issues.md

If ComfyUI starts successfully, most of these messages do not require action. Investigate logs only if ComfyUI fails to start, a workflow cannot run, or a plugin stops working.

## ComfyUI fails to start after upgrading to v1.0.37 or later

This issue may occur after upgrading to ComfyUI v1.0.37 or later.

> After upgrading ComfyUI, the app may fail to start and show an error like:

Reviewer: ```suggestion
After the upgrade, ComfyUI app may fail to start and show an error like:
```

```
main.py: error: unrecognized arguments: --normalvram
```

This means a custom launch argument from a previous version is still in use, but the new version no longer supports it.

To fix this:

1. Open **ComfyUI Launcher** and go to **Lab** from the left sidebar.

> 2. In the **Manually edit extra arguments** field, remove `--normalvram` manually and click **Save**. Alternatively, click **Restore default** to reset to the default launch arguments.

Reviewer: ```suggestion
2. In the **Manually edit extra arguments** field, remove `--normalvram` manually and click **SAVE MANUAL ARGS*. Alternatively, click **RESTORE DEFAULT** to reset to the default launch arguments.
```

> 2. In the **Manually edit extra arguments** field, remove `--normalvram` manually and click **SAVE MANUAL ARGS*. Alternatively, click **RESTORE DEFAULT** to reset to the default launch arguments.

Reviewer: ```suggestion
2. In the **Manually edit extra arguments** field, remove `--normalvram` manually and click **SAVE MANUAL ARGS**. Alternatively, click **RESTORE DEFAULT** to reset to the default launch arguments.
```

3. Verify that **Current full launch command** at the top no longer contains `--normalvram`.

> 4. In ComfyUI Launcher Home, click **Start** to launch the ComfyUI.

Reviewer: ```suggestion
4. Go back to **Home** in ComfyUI Launcher, then click **Start** to launch ComfyUI.
```
